packages/esm-appointments-app/translations/en.json

```diff
@@ -3,17 +3,13 @@
   "addNewClinicDay": "Add new clinic day",
   "anotherVisitType": "Start another visit type",
   "appointmentDateAndTime": "Appointments Date and Time",
-  "appointmentDateTime": "Appointment Date and Time",
   "appointmentForm": "Appointments Form",
   "appointmentFormError": "Error scheduling appointment",
   "appointmentHistory": "Appointment History",
   "appointmentList": "Appointments list",
   "appointmentMetrics": "Appointment metrics",
-  "appointmentNoteLabel": "Write an additional note",
-  "appointmentNotePlaceholder": "Write any additional points here",
```

Comment:
Some of these labels and placeholders were more verbose before. Do we really want to lose this verbage?

Reply:
I will maintain the wordings then.